### PR TITLE
[WIP] Version 2.0 Docs Updates

### PR DESCRIPTION
**Pull request by nniehoff**
This PR updates the documentation A LOT.  Not only content but v2 updates, as well as updating the format to be hosted on Read The Docs in the docs.nautobot.org format.  A LOT more coming...

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "nautobot-helm-charts"
+version = "0.1.0"
+description = "Helm Charts used to deploy Nautobot"
+authors = ["Network to Code, LLC <info@networktocode.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.11"
```
Comment (cmsirbu) [inline pyproject.toml:9]:
```suggestion
python = "^3.8"
```

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+for example:
+
+```console
+helm install nautobot nautobot/nautobot --set postgresql.auth.password="change-me" --set redis.auth.password="change-me"
+```
+
+## Support
+
+* **Slack**: checkout the **#nautobot** channel on the [Network to Code slack](https://networktocode.slack.com/)!
```
[the remark this answers is not in the document]
Comment (nniehoff) [reply docs/index.md:47]: @cmsirbu as this isn't the cookiecutter anymore, what is the correct value for `{{cookiecutter.docs_app_url}}`

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+---
```
[the remark this answers is not in the document]
Comment (nniehoff) [reply docs/index.md:1]: Still very much a work in progress, that's where I started... I'm hoping to not completely copy the readme

File: pyproject.toml (hunk 1 above)
Comment (cmsirbu) [reply pyproject.toml:9]: Pretty much this is why... you're limiting this project to only be developed on python 3.11.

```
The currently activated Python version 3.10.6 is not supported by the project (^3.11).
Trying to find and use a compatible version. 

Poetry was unable to find a compatible version. If you have one, you can explicitly use it via the "env use" command.
```

File: docs/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (cmsirbu) [reply docs/index.md:1]: This is *only* for the README. Inside the docs you definitely want to use relative links to md files.

File: docs/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (cmsirbu) [reply docs/index.md:47]: I think we can go with `https://docs.nautobot.com/projects/helm-charts/en/latest/` for this project.

File: docs/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nniehoff) [reply docs/index.md:1]: ok cool thanks for clarifying... I'm hoping I get the README down to `please see our docs page`

**Comment by nniehoff** [comment]
@cmsirbu a couple of questions:

How do we get the copy button in our code blocks?  (example https://docs.nautobot.com/projects/core/en/stable/installation/ubuntu/)

Any way we can get numbered lists to work right and continue from the previous numbers?  See https://helm-charts.readthedocs.io/en/latest/operations/upgrading/#example-upgrade-procedure and the raw source https://github.com/nautobot/helm-charts/blob/nn_next_docs/docs/operations/upgrading.md

**Comment by cmsirbu** [comment]
> @cmsirbu a couple of questions:
> 
> How do we get the copy button in our code blocks? (example https://docs.nautobot.com/projects/core/en/stable/installation/ubuntu/)

Looks like they added a toggle in 9.x: https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#code-copy-button (I will get the cookie updated with it).

> 
> Any way we can get numbered lists to work right and continue from the previous numbers? See https://helm-charts.readthedocs.io/en/latest/operations/upgrading/#example-upgrade-procedure and the raw source https://github.com/nautobot/helm-charts/blob/nn_next_docs/docs/operations/upgrading.md

Markdown is very finicky with "interrupted" numbered lists, but I find that if you indent the stuff between list items at 4 spaces it works (at least in github markdown and mkdocs). 

> 
> Check the links https://helm-charts.readthedocs.io/en/latest/advanced-features/ I'm not sure how to fix them considering they are all broken

Links work fine for me, can you please clarify what's broken in there?

**Pull request merged**